lib/applicant_factory.rb

```diff
@@ -23,7 +23,7 @@ def make_a_person(applicant)
         work_phone: Faker::PhoneNumber.phone_number,
         home_phone: Faker::PhoneNumber.phone_number,
         cell_phone: Faker::PhoneNumber.phone_number,
-        citizenship: Faker::Address.country,
+        citizenship: ["US Citizen", "Non-Citizen with eligible immigration status", "Other"].sample,
```

Comment:
You'll probably want to update Person#us_citizen? too, because right now it does this:

````
   def us_citizen?
     return "United States" == citizenship
   end
````